docs/acton-by-example/src/types.md

```diff
@@ -2,6 +2,40 @@
 
 Acton is a statically typed language. Unlike traditional languages like C or Java, where the type of every variable must be explicitly stated, we can write most Acton programs without types. The Acton compiler features a powerful type inferencer which will then infer the type.
 
+To see the inferred type signatures after an Acton program is compiled you can use the `--sigs` option of the compiler. As rhe name suggests, this will print out the type signatures for the functions, actors and their attributes and methods found in the compiled file.
+
+Source:
+```python
+def foo(a, b):
+    if a > 4:
+        print(len(b))
+
+actor main(env):
+    i1 = 1234
+    s1 = "hello"
+    foo(i1, s1)
+
+    await async env.exit(0)
+```
+
+Compile and use the `--sigs` option:
+```sh
+actonc --sigs types.act
+Building file types.act
+  Compiling types.act for release
+
+
```

Comment:
```suggestion
actonc types.act --sigs

```

Freshest actonc doesn't print the "Building file" cruft for alternate outputs like --sigs...

Also provide file first. Actonc has multiple commands and a file can be thought of as an alias for "just build this file". --sigs is relevant for a single file, but it is not relevant to for example `actonc new my-new-project`. We should encourage users to think of "first command then followed by relevant arguments"